frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "@angular/platform-browser": "~2.0.0",
     "@angular/platform-browser-dynamic": "~2.0.0",
     "@angular/router": "~3.0.0",
+    "angular2-mdl": "^2.3.1",
```

Comment:
@sneridagh we should try to avoid using ^. I know this is the default if you do a "--save". Maybe we can add a check to travis for that...